operations/System_Recovery/PBS_Service_Recovery.md

```diff
@@ -10,9 +10,8 @@ The following covers restoring PBS data.
 
 ## Service recovery for PBS
 
-To restore PBS Professional data from backup, follow section
-*10.7.7 Restore PBS home directory from backup* in the
-**HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX Systems (S-8003)**.
+To restore Portable Batch System data from backup, see *Restore PBS home directory from backup* in the
```

Comment:
```suggestion
To restore Portable Batch System data from a backup, see *Restore PBS home directory from a backup* in the
```

Reply:
Added a few minor changes in the CSM PRs I'm working on.

Reply:
I'm basing that string on the section name in the actual doc, so I think leaving it w/o the added " a " is best.

Reply:
Here's the PR with the added "a" in the section titles of the document: https://github.hpe.com/hpe/hpc-wlm-pbs-docs/pull/31. I'll likely be merging this PR today. A PDF version is attached for reference.

[PBS_CSM_Install_Guide_revised.pdf](https://github.com/Cray-HPE/docs-csm/files/14388711/PBS_CSM_Install_Guide_revised.pdf)